saw-core/src/Verifier/SAW.hs

```diff
@@ -22,5 +22,8 @@ import Verifier.SAW.Prelude
 import Verifier.SAW.ExternalFormat
 
 -- The following type-checks the Prelude at compile time, as a sanity check
-import Language.Haskell.TH
-$(runIO (mkSharedContext >>= \sc -> scLoadPreludeModule sc >> return []))
+-- NOTE: this is now done in Verifier.SAW.Cryptol, which also type-checks the
+-- Cryptol-related SAW core modules as well
+--
+-- import Language.Haskell.TH
+-- $(runIO (mkSharedContext >>= \sc -> scLoadPreludeModule sc >> return []))
```

Comment:
Perhaps this comment should just be removed, as it's a historical note that no longer applies to the current version of the code.